src/main/java/JsonHelper.java

```diff
@@ -30,6 +30,7 @@ public TranslationRequestBody(String q, String sourceLanguage, String target) {
             this.target = target;
         }
 
+        //getters and setters are never used, but are required for jackson
```

Comment:
i feel like you deleted them, figured out it doesn't work anymore and added them back with the comment... i did that before too